exchange/binance.go

```diff
@@ -23,6 +23,7 @@ type Binance struct {
 	client     *binance.Client
 	assetsInfo map[string]model.AssetInfo
 	userInfo   UserInfo
+	HeikinAshi bool `default:"false"`
```

Comment:
The tag is not necessary
```suggestion
	HeikinAshi bool
```